### DISCUSSION TITLE
Option to View Default Value of a Setting

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1041] [Ideas]
Add an option that allows you to hover over a changed setting and monkeytype will display the default value of it

**Comment by Miodec** [comment]
Sure, ill add it to the descriptions in the settings page.

**Reply by Miodec** [reply]
So, im adding the defaults right now and realised pretty much 95% of them are default off. Im not sure it will provide anything meaningful.

**Reply by unrealapex** [reply]
In the command line, I think something like this would be good enough(default indicator is italic): 
<details>
<summary>Font size...</summary>
<ul>
<li>1x</li>
<li>1.25x</li>
<li>1.5x <i>(default)</li>
<li>2x</li>
<li>3x</li>
<li>4x</li>
</ul>
</details>
